db/comdb2.c

```diff
@@ -4559,7 +4559,7 @@ void *statthd(void *p)
                     unsigned nwaits = cur_bdb_stats.n_lock_waits -
                                       last_bdb_stats.n_lock_waits;
                     reqlog_logf(statlogger, REQL_INFO,
-                                "%u locks, avg time %ums, worst time %ums\n",
+                                "%u locks, avg time %ums, worst time=%ums\n",
```

Comment:
Why do we need the `=` here?

Reply:
leftover from other related work--removing. thanks